README.md

```diff
@@ -135,18 +135,4 @@ docker run gcr.io/gcp-variant-transforms/gcp-variant-transforms:latest \
 ```
 
 ## Development
-
-### Testing
-
-To run all tests:
-
-```bash
-python setup.py test
-```
-
-To run a specific test:
-
-```bash
-python setup.py test -s <module>.<test class>.<test method>
-```
-
+[Development Guide](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/docs/development_guide.md)
```

Comment:
please use relative path here (and in the file below).

Reply:
Done.